src/flamegpu/sim/SimRunner.cu

```diff
@@ -104,7 +106,7 @@ void SimRunner::start() {
             }
             log_export_queue_cdn.notify_one();
             // Print progress to console
-            if (verbose) {
+            if (verbosity == VERBOSE) {
```

Comment:
This should be "Default" rather than "Verbose" imo.

It' uses `\r` so it just rewrites the same line of output to show progress.
(However, it will write something ugly if output is being piped to file as we don't detect to provide an alt output).

Reply:
I'm almost certain that previously always progress printed by default. It's very useful to know how far along your huge batch ensemble is with processing.

Reply:
Yes, `verbose` would be set `true` unless quiet was enabled.

https://github.com/FLAMEGPU/FLAMEGPU2/blob/0f5a4c597560a6991a9a9b33c730fb9fc18622ad/src/flamegpu/gpu/CUDAEnsemble.cu#L148

Reply:
The outputs should not be "racy", we fixed the issue where progress counter would briefly count backwards (it was printing the id of the last finished job, rather than finished count).

Reply:
I'm personally inclined to say Sim's should always be marked Quiet by ensembles. Their output is kinda useless if it's not associated with a particular run during concurrent ensembles.

Reply:
To achieve what you want, would be to add a programtic option to CUDASimulation which pipes stdout/stderr to file. So a user can review them in isolation after.